connectivity/FEATURE_BLE/source/generic/GapImpl.cpp

```diff
@@ -1757,7 +1661,7 @@ ble_error_t Gap::setExtendedAdvertisingParameters(
         params.getChannel39()
     );
 
-    ble_error_t err = _pal_gap.set_extended_advertising_parameters(
+    auto err = _pal_gap.set_extended_advertising_parameters(
```

Comment:
I hope this is not a sign of things to come where every type that can be changed to auto will be